its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -202,8 +198,7 @@ public void testHelpMessage() throws IOException {
     Assume.assumeTrue(TestUtils.getScannerVersion(ORCHESTRATOR) == null);
 
     Path projectDir = TestUtils.projectDir(temp, "ProjectUnderTest");
-    BuildResult result = ORCHESTRATOR.executeBuild(TestUtils.newScanner(ORCHESTRATOR, projectDir)
-      .addArgument("/?"));
+    BuildResult result = ORCHESTRATOR.executeBuild(ScannerForMSBuild.create(projectDir.toFile()).addArgument("/?"));
```

Comment:
Educational: How come you changed the method call completely? (from `TestUtils.newScanner` to `ScannerForMSBuild.create`)

I understand that this test just verifies that the help message behaves as expected, is this just a more lightweight way of instantiating a `ScannerForMsBuild`?

Reply:
Yes. The `TestUtils.newScanner` is adding a couple of parameters, and now the token, and this test should check only the help.